File: fixed_stack_dynamic.cpp
#include <iostream>
using namespace std;

int main() {
    // Fixed size array allocated on stack at compile time
    int arr[5] = {1, 2, 3, 4, 5};  // Array size is fixed and known at compile time
    for (int i = 0; i < 5; i++) {
        cout << arr[i] << " ";  // Output the array values
    }
    return 0;
}

File: fixed_stack_dynamic.js
// JavaScript does not have stack-allocated arrays with fixed sizes.
// Simulating fixed-size array with fill and const
const arr = [];  // Create an empty array
for (let i = 0; i < 5; i++) {
    arr[i] = i + 1;  // Fill the array with values 1 to 5
}
console.log(arr.join(" "));  // Print array values separated by space
